codecov_auth/tests/unit/views/test_okta_mixin.py

```diff
@@ -0,0 +1,110 @@
+from unittest.mock import MagicMock, patch
+
+import jwt
+
+from codecov_auth.views.okta import OKTA_BASIC_AUTH, OktaLoginView
+from codecov_auth.views.okta_mixin import validate_id_token
+
+private_key = """-----BEGIN PRIVATE KEY-----
```

Comment:
Is this a legit private key? I doubt it is, but would you be able to name this something like "fake_private_key" or something? We get pinged at times with potential security vulns that see things like this make it to prod, so this would make it very obvious if the security team or someone asks in retrospect